book/SUMMARY.md

```
@@ -33,7 +33,9 @@
   - [strace](./comparison/strace.md)
 - [FAQ](./faq.md)
 - [Developer Guide](./dev.md)
-  - [Internal Architecture]()
+  - [Internal Architecture](./dev/architecture.md)
   - [Backend Differences]()
```

markdownlint-cli2 (0.22.1)

[warning] No empty links

(MD042, no-empty-links)